src/plugins/gui/providers/banxaProvider.ts

```diff
@@ -273,12 +353,10 @@ export const banxaProvider: FiatProviderFactory = {
         return allowedCurrencyCodes[direction]
       },
       getQuote: async (params: FiatProviderGetQuoteParams): Promise<FiatProviderQuote> => {
-        const { pluginId, regionCode, exchangeAmount, amountType, paymentTypes, fiatCurrencyCode, displayCurrencyCode, direction } = params
+        const { pluginId, regionCode, exchangeAmount, amountType, paymentTypes, fiatCurrencyCode, displayCurrencyCode, direction, tokenId } = params
         if (!paymentTypes.some(paymentType => allowedPaymentTypes[direction][paymentType] === true))
           throw new FiatProviderError({ providerId, errorType: 'paymentUnsupported' })
```

Comment:
Now that we have buy + sell, sounds like a good time to add `direction` to these errors.

Reply:
We could but the parent plugin already knows the direction so it should be able to imply that

Reply:
Yeah that's true as it's designed now, and the design reads like there's some possibility that the error info wouldn't match the direction param in getBestError. 

The error would be more versatile if that more info just came attached to the error. Then the consumers of the error (analytics, bugsnag) wouldn't need to know about the implementation of the feature to fully report the full context of the error.